### PR TITLE
feature/SM-4128: Inspection Units and Final Reinstatements

### PR DESCRIPTION
**Pull request by ccurran01**
Update inspection units endpoint documenation
Add final reinstatement endpoint documentation as this field has been removed from the excavation endpoint
Remove references to the excavation carried out endpoint from the documentation

### CODE REVIEW

File: docs/api-documentation/V1.10/index.md [hunk 1]
```diff
@@ -1770,26 +1764,26 @@ By setting an actual stop date via this endpoint, the active permit's status wil
...
 
-#### Excavation carried out endpoint
+#### Inspection units endpoint
 {: .govuk-heading-s}
 
-<code>PUT /works/{work reference number}/excavation</code>
+<code>PUT /works/{work reference number}/inspection-units</code>
 
-When a permit is submitted initially the excavation property indicates whether or not an excavation will need to be carried out as part of the work. Similar to proposed start and stop, this endpoint indicates whether or not an excavation was *actually* carried out.
+Once a permit is in progress and an excavation has been carried out, a promoter can log the inspection units to the Works Record.
```
Comment (matthewdoreilly) [inline docs/api-documentation/V1.10/index.md]:
```suggestion
Once a permit is in progress, and an excavation has been carried out, a promoter can log the inspection units to the Works Record.
```

File: docs/api-documentation/V1.10/index.md [hunk 2]
```diff
@@ -1770,26 +1764,26 @@ By setting an actual stop date via this endpoint, the active permit's status wil
...
 {: .govuk-body}
 
-Marking an excavation was carried out on an in-progress work record allows promoters to add reinstatements to that works record. Once a reinstatement exists you can no longer use this endpoint to change whether or not an excavation was carried out.
+You can only provide inspection-units if a reinstatement currently exists on the works record, which will cover in a separate section.
```
Comment (matthewdoreilly) [inline docs/api-documentation/V1.10/index.md]:
```suggestion
You can only provide inspection units if a reinstatement currently exists on the works record, which will cover in a separate section.
```

Comment (matthewdoreilly) [reply docs/api-documentation/V1.10/index.md]: Also what do you mean by "which will cover in a separate section"?

File: docs/api-documentation/V1.10/index.md [hunk 3]
```diff
@@ -1770,26 +1764,26 @@ By setting an actual stop date via this endpoint, the active permit's status wil
...
 
-<code>PUT /works/{work reference number}/inspection-units</code>
+<code>PUT /works/{work reference number}/final-reinstatement</code>
 
-Once a permit is in progress and an excavation has been carried out, a promoter can log the inspection units associated with the currently active permit.
+Once a permit is in progress, and an excavation site has been added to the work, a promoter can flag that the final site has been registered for that work.
 {: .govuk-body}
 
-You can only provide inspection-units if a reinstatement currently exists on the works record, which will cover in a separate section.
+You can only provide final-reinstatement if an excavation reinstatement currently exists on the works record, which is covered in a separate section.
```
Comment (matthewdoreilly) [inline docs/api-documentation/V1.10/index.md]:
```suggestion
You can only provide final reinstatement if an excavation reinstatement currently exists on the works record, which is covered in a separate section.
```

File: docs/api-documentation/V1.10/index.md (hunk 2 above)
Comment (shaunganley) [inline docs/api-documentation/V1.10/index.md]:
```suggestion
You can only provide inspection-units if a reinstatement currently exists on the works record, which is covered in a separate section.
```?

**Comment by ccurran01** [comment]
> Just some small comments. Also, can you add what has changed to the API docs to the `Versions and Changes` section at the bottom?

Was added by PK. Just some other updates so the docs are closer in line with the works

**Pull request merged**